cql2pgjson/src/main/java/org/folio/cql2pgjson/util/DbSchemaUtils.java

```diff
@@ -54,4 +65,99 @@ public static Index getIndex(String cqlIndex, List<Index> indexes) {
     return null;
   }
 
+  /**
+   * Get {@link Table} for given table name in {@link Schema}
+   *
+   * @param dbSchema
+   * @param tableName
+   * @return
+   */
+  public static Table getTable(Schema dbSchema, String tableName) {
+    for (Table table : dbSchema.getTables()) {
+      if (tableName.equals(table.getTableName())) {
+        return table;
+      }
+    }
+    return null;
+  }
+
+  /**
+   * Find a list of {@link ForeignKeys} from source table to target table alias
+   *
+   * @param dbSchema
+   * @param srcTabName
+   * @param targetTabAlias
+   *
+   * @return a list of {@link ForeignKeys}
```

Comment:
Can you document what is returned if not found?

Reply:
done